packages/@ourworldindata/utils/src/regions.ts

```diff
@@ -0,0 +1,70 @@
+import entities from "./regions.json"
+
+export interface Country {
+    code: string
+    shortCode?: string
+    name: string
+    shortName?: string
+    slug: string
+    regionType?: "country" | "other"
+    isMappable?: boolean
+    isHistorical?: boolean
+    isUnlisted?: boolean
+    variantNames?: string[]
+}
+
+export interface Aggregate {
+    name: string
+    regionType: "aggregate"
+    code: string
+    members: string[]
+}
+
+export interface Continent {
+    name:
+        | "Africa"
+        | "Asia"
+        | "Europe"
+        | "North America"
+        | "Oceania"
+        | "South America"
+    regionType: "continent"
+    code: string
+    members: string[]
+}
+
+export type Region = Country | Aggregate | Continent
+
+export const regions: Region[] = entities as Region[]
+
+export const countries: Country[] = regions.filter(
+    (entity: any) =>
+        entity.regionType === "country" &&
+        !entity.isUnlisted &&
+        !entity.isHistorical
+) as Country[]
+
+export const others: Country[] = entities.filter(
+    (entity: any) => entity.regionType === "other"
+) as Country[]
```

Comment:
```suggestion
export const others: Country[] = regions.filter(
    (entity: Region) => entity.regionType === "other"
) as Country[]
```

same for the others above and below